documentation/book/assembly-kafka-mirror-maker-configuration.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-configuration-{context}']
+
+= Kafka Mirror Maker configuration
+
+{ProductName} allows you to customize the configuration of Apache Kafka Mirror Maker by editing most of the options for the related consumer and producer.
+Producer options are listed in {ApachaKafkaProducerConfig}.
+Consumer options are listed in {ApachaKafkaConsumerConfig}.
+
+The only options which cannot be configured are those related to the following areas:
+
+* Kafka cluster bootstrap address
+* Security (Encryption, Authentication, and Authorization)
+* Consumer group identifier
```

Comment:
I think you should be explicit about which config option prefixes are forbidden. I think that's what we do for the forbidden broker configs, for example.